installer/assets/THIRD-PARTY-LICENSES

```diff
@@ -670,7 +670,7 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 The above BSD License Applies to all code, even that also covered by Apache 2.0.
 
 --------------------------------------------------------------------------------
-** pygments; version 2.15.0 -- https://pypi.org/project/pygemnts/
+** pygments; version 2.15.0 -- https://pypi.org/project/Pygments/
```

Comment:
```suggestion
** pygments; version 2.18.0 -- https://pypi.org/project/Pygments/
```

Reply:
The version number should actually be 2.18.0 as seen here https://github.com/aws/aws-sam-cli/blob/d5fd4b803b6a0a9db46ad5102bf0eae8176e635e/requirements/dev.txt#L18